core/lib/multivm/src/versions/vm_refunds_enhancement/tracers/dispatcher.rs

```diff
@@ -13,6 +13,12 @@ pub struct TracerDispatcher<S: WriteStorage, H: HistoryMode> {
     tracers: Vec<TracerPointer<S, H>>,
 }
 
+impl<S: WriteStorage, H: HistoryMode> TracerDispatcher<S, H> {
+    pub fn new(tracers: Vec<TracerPointer<S, H>>) -> Self {
```

Comment:
Off-topic: Maybe, it's personal, but using `_Pointer` suffix for a boxed trait object looks *bizarre* to me (yeah, I understand that `Box` is a smart pointer). `Boxed_` / `Box_` would look much more like a conventional name; cf. [`BoxFuture`](https://docs.rs/futures/latest/futures/future/type.BoxFuture.html) from the `futures` crate.

Reply:
So, actually here i want to keep a bit flexibility and play with different smart pointers. E.g. Rc<RefCell>. Initially it was boxed, as you suggest.